bindings/matrix-sdk-crypto-js/src/requests.rs

```diff
@@ -36,15 +36,15 @@ pub struct KeysUploadRequest {
     /// A JSON-encoded string containing the rest of the payload: `device_keys`,
     /// `one_time_keys`, `fallback_keys`.
     #[wasm_bindgen(readonly)]
-    pub body: JsString,
+    pub extra: JsString,
```

Comment:
given that this corresponds exactly to the body of the HTTP request, `body` seems like a good name for it. YMMV though.

Reply:
OK done :-), let's keep `body`.